components/library/webm/libvpx/Makefile

```diff
@@ -0,0 +1,62 @@
+#
```

Comment:
They are part of the same project family. OI-Userland tends to group those (e.g. samba of various versions, docbook-related stuff, etc.)

It would also make the directory structure maybe easier to navigate (less entries in each dir) if we choose to keep several complete versions of the package delivered (e.g. not only .so files, but also includes, etc.)